bitcoinspv/types/btccache.go

```diff
@@ -8,9 +8,9 @@ import (
 
 // BTCCache represents a thread-safe cache of indexed blocks
 type BTCCache struct {
+	blocks []*IndexedBlock
 	sync.RWMutex
```

Comment:
Inherited type must be first. If the winter is complaining, then let's disable it